### PR TITLE
Update object string dumps

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
With the scenario text PR in, this PR updates the object string dumps for all languages, sorted by object id.

This PR as a whole has a pretty terrible diff, as the previous dump was apparently not ordered by id. The first commit orders the files in-tree by object identifier key, while the second commit brings the dumps up-to-date.

### CODE REVIEW

**Comment by Gymnasiast** [comment]
I want to be able to see if this doesn’t accidentally overwrite any existing translations.

I want to do that as follows:
- Take the existing JSON files
- Sort them
- Merge it
- Rebase this on top of that.

**Comment by AaronVanGeffen** [comment]
I can easily do that (`jq --sort-keys`) and just regenerate the dumps, if you like.

**Comment by Gymnasiast** [comment]
Yes, please.

**Comment by Gymnasiast** [comment]
Could you re-dump to account for the changed IDs?

**Comment by AaronVanGeffen** [comment]
Updated the dumps using the branch for https://github.com/OpenRCT2/objects/pull/353.

**Comment by Gymnasiast** [comment]
Apologies, I already merged #2966 before I saw your request.

**Comment by AaronVanGeffen** [comment]
Thankfully, there doesn't seem to be a conflict. Phew! I'm just glad I don't have to do it over again 😄

File: objects/pl-PL.json [hunk 1]
```diff
@@ -783,8 +1495,8 @@
     "name": "Drewno",
     "reference-name": "Wood"
   },
-  "rct2.audio.base": {
-    "name": "Bazowe efekty dzwiękowe gry.",
+  "rct2.audio.base.rct2": {
+    "name": "Base audio for the game.",
```
Comment (Gymnasiast) [inline objects/pl-PL.json:1499]:
```suggestion
    "name": "Bazowe efekty dzwiękowe gry.",
```

File: objects/pl-PL.json [hunk 2]
```diff
@@ -4863,7 +5803,7 @@
     "capacity": "2 pasażerów na wagonik",
```
Comment (Gymnasiast) [inline objects/pl-PL.json:5803]: Incorrect.

File: objects/pl-PL.json [hunk 3]
```diff
@@ -5013,7 +5953,7 @@
     "capacity": "10 pasażerów na wagonik",
```
Comment (Gymnasiast) [inline objects/pl-PL.json:5953]: Incorrect.

File: objects/pl-PL.json (hunk 2 above)
Comment (Gymnasiast) [inline objects/pl-PL.json:5803]:
```suggestion
    "capacity": "1 pasażer na wagonik",
```

File: objects/de-DE.json [hunk 4]
```diff
@@ -4750,7 +5690,7 @@
   "rct2tt.ride.cyclopsx": {
     "capacity": "1 Person pro Wagen",
     "description": "Die Fahrer fahren im Auge eines Riesenzyklopen",
-    "name": "Zyklopen-Autoscooter",
+    "name": "Zyklopenfahrt",
```
Comment (Gymnasiast) [inline objects/de-DE.json:5693]: check

Comment (Gymnasiast) [reply objects/de-DE.json:5693]:
```suggestion
    "name": "Zyklopen-Autoscooter",
```

File: objects/de-DE.json [hunk 5]
```diff
@@ -4790,7 +5730,7 @@
   "rct2tt.ride.flwrpowr": {
     "capacity": "1 Fahrgast pro Wagen",
     "description": "Die Fahrer fahren in blumenförmigen Luftkissenfahrzeugen, die sie selbst steuern",
-    "name": "Flower-Power-Autoscooter",
+    "name": "Flower-Power-Fahrt",
```
Comment (Gymnasiast) [inline objects/de-DE.json:5733]:
```suggestion
    "name": "Flower-Power-Autoscooter",
```

File: objects/de-DE.json [hunk 6]
```diff
@@ -7474,7 +8526,7 @@
   "rct2ww.ride.dragdodg": {
     "capacity": "1 Fahrgast pro Wagen",
     "description": "Die Gäste liefern sich Gefechte in drachenkopfförmigen Autoscootern",
-    "name": "China-Drachenkopf-Autoscooter",
+    "name": "Chinesische Drachenkopffahrt",
```
Comment (Gymnasiast) [inline objects/de-DE.json:8529]:
```suggestion
    "name": "China-Drachenkopf-Autoscooter",
```

File: objects/fi-FI.json [hunk 7]
```diff
@@ -28,41 +28,49 @@
     "reference-name": "Support Structure"
   },
   "official.terrain_edge.void": {
-    "name": "Tyhjä",
+    "name": "Void",
```
Comment (Gymnasiast) [inline objects/fi-FI.json:31]: You are mass-overwriting Finnish translations with English here. That can’t be right.

File: objects/zh-CN.json [hunk 8]
```diff
@@ -440,15 +632,15 @@
     "reference-name": "Ash Footpath (Square)"
   },
   "rct1aa.footpath_surface.queue_green": {
-    "name": "绿色队列",
+    "name": "Green Queue",
```
Comment (Gymnasiast) [inline objects/zh-CN.json:635]: Also needs syncing.

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**